packages/core/src/browser/storage-service.ts

```diff
@@ -45,10 +46,13 @@ export class LocalStorageService implements StorageService {
     private storage: LocalStorage;
 
     constructor(
-        @inject(ILogger) protected logger: ILogger
+        @inject(ILogger) protected logger: ILogger,
+        @inject(MessageService) protected readonly messageService: MessageService,
+
```

Comment:
Use property injection rather than constructor injection for new dependencies.

Reply:
Still using the constructor injection. If I use the property injection, the front-end does not start and report the messageServide as undefined, so kept it with the constructor's injection.

Reply:
that's because properties are injected after constructor call. please use `@postConstruct protected init() { }` for initialization relying on injected dependencies.